lightstep/constants.py

```diff
@@ -24,3 +24,7 @@
 
 # utils constants
 SECONDS_TO_MICRO = 1000000
+
+# Recorder constants
+MAX_LOG_MESSAGE_LEN = 1024
+LONGEST_POSSIBLE_SUB_LOG = 984
```

Comment:
why are both of the above needed? (why don't we just truncate at MAX_LOG_MESSAGE_LEN?) sorry if I'm missing something obvious...